docs/README.md

```diff
@@ -1,16 +1,16 @@
 # Splunk Enterprise Docker Container
 
-**Splunk's Docker container for Splunk Enterprise and Splunk Universal Forwarder are designed to help first-time users quickly deploy and gain hands-on experience with the Splunk software, while still allowing for complex deployments in the future**
+**Use the Docker containers for Splunk Enterprise and the Splunk Universal Forwarder to quickly deploy Splunk software, with the ability to add complexity in the future.**
```

Comment:
Open to changes here, if this isn't the primary use case we want to get across. The original lacked concision and clarity.